src/views/RessourcesView.vue

```diff
@@ -1,29 +1,31 @@
 <script setup lang="ts">
+import codeLanguages from '@clubcapra/data/GithubStats.json';
+
 // Components
 import BibBaseComponent from '@clubcapra/components/BibBaseComponent.vue';
 import JumbotronVideoComponent from '@clubcapra/components/JumbotronVideoComponent.vue';
 
 // Media
 import codebg from '@clubcapra/assets/media/codebg.jpg';
 
-const codeLanguages = [
-  {
-    name: 'C++',
-    percent: 70,
-  },
-  {
-    name: 'TypeScript',
-    percent: 20,
-  },
-  {
-    name: 'Python',
-    percent: 5,
-  },
-  {
-    name: 'C',
-    percent: 5,
-  },
-];
+// const codeLanguages = [
+//   {
+//     name: 'C++',
+//     percent: 70,
+//   },
+//   {
+//     name: 'TypeScript',
+//     percent: 20,
+//   },
+//   {
+//     name: 'Python',
+//     percent: 5,
+//   },
+//   {
+//     name: 'C',
+//     percent: 5,
+//   },
+// ];
```

Comment:
Don't leave commented-out code. Git already enables code retrieval from the version history.